integration/107_launch_with_ebpf_test.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+# shellcheck disable=SC1091
+. ./config.sh
+
+start_suite "Launch scope and check it boots, with ebpf connection tracking enabled"
```

Comment:
This test is not all that useful. It simply checks that the container is created. It may as well be crashing after bootup.

I would say, either make it more useful or remove it.